### PR TITLE
Issue #42

### PR DESCRIPTION
**Pull request by bubblecode76**
updated the format of the cheatsheet

### CODE REVIEW

**Review by Aatmaj-Zephyr — APPROVED** [review]
Nice changes

**Comment by EshanTrivedi21** [comment]
Great work @bubblecode76 !! 
Will merge it in a while after solving the merge conflicts!

**Pull request merged**
